kernel/configs/arm64_kata_kvm_4.14.x

```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/arm64 4.14.72 Kernel Configuration
+# Linux/arm64 4.14.67 Kernel Configuration
```

Comment:
slightly odd having the kernel version go down - I guess you are testing on a kernel not quite as recent as the original - I'll let you decide if that is right or not ;-)

Reply:
@grahamwhaley we change version from 72 to 67 according to the kernel version setting in runtime/versions.yaml. 
as kernel version in https://cdn.kernel.org/pub/linux/kernel/v4.x/ is upper than 67, maybe we can upgrade it in runtime/version.yaml.

Reply:
Yeah, we have 67 in our versions.yaml file. Should be in sync.